src/main/java/seedu/address/logic/commands/TagCommand.java

```diff
@@ -94,6 +100,24 @@ public CommandResult execute(Model model) throws CommandException {
         return new CommandResult(finalMessage.toString());
     }
 
+    /**
+     * Undoes the previous Tag command
+     */
+    public void undo(Model model) {
```

Comment:
For better readability and clarity, an @Override annotation can be placed here to indicate that you are overriding from the interface.